elastic/kibana
# Gracefully handle Not Found error in ruleClient.getAlertState

Resolves: #175617

## To verify:

1- replace `rule.scheduledTaskId` with a random string in: https://github.com/elastic/kibana/pull/177077/files#diff-5f4b0a0f0ee693873ec9d82c4fa713b9412dad9075bc7fc86ee7b8c9c4efe01bR32

2- create a rule 

3- use dev tools to get the rule state with :  `GET kbn:/internal/alerting/rule/{your-rule-id}/state`

response should be `204 no content` and there should be a message in the console like: `Task (your-rule-id) not found`
